apertus-open-source-cinema/axiom-beta-firmware
# mainline uboot

use mainline u-boot instead of the xilinx fork

Todo:
- [x] test on micro
- [x] test on beta
- [x] make qemu working again
   caveat: qemu boots the kernel directly and does not boot `u-boot` for now (until we can figure out how to make `u-boot` work again)

Maybe we also want to use falcon mode, but probably not (would boot a bit faster, but we loose all the nice u-boot features like tftp boot.

This does some minor build system rework, to
- add patch support for u-boot
- add device specific configuration (we might want to integrate some existing stuff into that)
- use `.install_stamp` for `boot.fs` aswell, as that should work better
